### DISCUSSION TITLE
How can I add to cmp sources?

### DISCUSSION THREAD

**Discussion by ghost** [LunarVim/LunarVim#1927] [Q&A] [answered]
Is there a way to add to LunarvVm's list of cmp sources, without overriding what is already there?

Say I want to add this.

```lua
require'cmp'.setup({
  sources = {
    { name = 'orgmode' }
  }
})
```

This seems to override all the previous sources, I just want to add to what is already there. Thanks!

**Comment by aaronclimbs** [comment, marked as answer]
I'm pretty sure the correct syntax here would be 
```lua
table.insert(lvim.builtin.cmp.sources, { name = "orgmode" })
```

**Reply by ghost** [reply]
@aaronclimbs This works perfectly, thank you!
